CHANGES.md

```diff
@@ -1,6 +1,7 @@
 # UNRELEASED
 - [NEW] Allow a custom HTTP agent to be specified.
 - [NEW] Trim the database URL if it includes the `/_changes` endpoint.
+- [FIXED] Correctly pass request parameters to feed filter function.
```

Comment:
How about we mark this up as `feed.filter` so we distinguish it from the `.../_changes?filter=`

Reply:
Yup. Will amend.